dotnet/samples/HomeAutomation/Program.cs

```diff
@@ -0,0 +1,93 @@
+﻿/*
+ Copyright (c) Microsoft. All rights reserved.
+
+ Example application that uses Semantic Kernel.
+
+ Loads app configuration from:
+ - appsettings.json.
+ - appsettings.{Environment}.json.
+ - Secret Manager when the app runs in the "Development" environment (set through the DOTNET_ENVIRONMENT variable).
+ - Environment variables.
+ - Command-line arguments.
+*/
+
+using HomeAutomation.Plugins;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.OpenAI;
+
+namespace HomeAutomation;
+
+internal static class Program
+{
+    internal static async Task Main(string[] args)
+    {
+        HostApplicationBuilder builder = Host.CreateApplicationBuilder(args);
+        builder.Services.AddHostedService<Worker>();
+        builder.Services.AddSingleton<AppConfiguration>();
+        builder.Services.AddSingleton<Kernel>(sp =>
```

Comment:
Just realized I mispoke when I sent a message to you earlier... the kernel should probably a transient; I think the services _underneath_ it, should be singletons

Reply:
Done